docs/components/slideout.md

```diff
@@ -64,9 +64,214 @@ Below we demonstrate wrapping `KSlideout` in the [`KToggle`](/components/renderl
+    <KSlideout :is-visible="isToggled.value" @close="toggle" :overlay-enabled="false" close-button-alignment="end" :has-only-title="true" title="I only have a Title">
+      <div>
+        <h2>Not only can you put any html in here like the paragraph below but you can also use other components</h2>
+        <p>Anim officia eiusmod duis est consequat nulla tempor ad non magna Lorem ullamco nostrud amet. Occaecat voluptate dolor enim eiusmod do qui nulla pariatur enim. Et elit elit consequat do do duis enim est ullamco id sunt sunt amet eiusmod. Do minim mollit irure ea sunt officia minim sint eiusmod enim amet. Quis exercitation in ullamco quis aliqua.</p>
+        <p>Anim officia eiusmod duis est consequat nulla tempor ad non magna Lorem ullamco nostrud amet. Occaecat voluptate dolor enim eiusmod do qui nulla pariatur enim. Et elit elit consequat do do duis enim est ullamco id sunt sunt amet eiusmod. Do minim mollit irure ea sunt officia minim sint eiusmod enim amet. Quis exercitation in ullamco quis aliqua.</p>
+        <p>Anim officia eiusmod duis est consequat nulla tempor ad non magna Lorem ullamco nostrud amet. Occaecat voluptate dolor enim eiusmod do qui nulla pariatur enim. Et elit elit consequat do do duis enim est ullamco id sunt sunt amet eiusmod. Do minim mollit irure ea sunt officia minim sint eiusmod enim amet. Quis exercitation in ullamco quis aliqua.</p>
+        <p>Anim officia eiusmod duis est consequat nulla tempor ad non magna Lorem ullamco nostrud amet. Occaecat voluptate dolor enim eiusmod do qui nulla pariatur enim. Et elit elit consequat do do duis enim est ullamco id sunt sunt amet eiusmod. Do minim mollit irure ea sunt officia minim sint eiusmod enim amet. Quis exercitation in ullamco quis aliqua.</p>
+        <p>Anim officia eiusmod duis est consequat nulla tempor ad non magna Lorem ullamco nostrud amet. Occaecat voluptate dolor enim eiusmod do qui nulla pariatur enim. Et elit elit consequat do do duis enim est ullamco id sunt sunt amet eiusmod. Do minim mollit irure ea sunt officia minim sint eiusmod enim amet. Quis exercitation in ullamco quis aliqua.</p>
+      </div>
+      <KButton appearance="primary">Buttons</KButton>
+      <KCard class="mt-2">
+        <template v-slot:body>
+          Something in a KCard
+        </template>
+      </KCard>
+      <KAlert
+        appearance="info"
+        alert-message="Or even an alert!"
+        class="mt-2" />
+    </KSlideout>
+  </div>
+</KToggle>
+```
+
+### badgeValue
+
+This prop takes a string that will be displayed as the value for the `Badge`.
+
+### badgeAppearance
+
+Determines the appearance values for the Badge, defaults to `warning`.
+
+### badgeShape
+
+Determines the shape of the Badge, defaults to `rectangular`.
+
+### iconType
+
+Determines the icon type that is placed next to Title, defaults to `copy`.
+
+### iconColor
+
+Determines the icon color that is placed next to Title, defaults to `blue`.
+
+### iconSize
```

Comment:
issue: We are actually trying to move away from this type of approach with too many props customizing optional parts of the design. It would be much cleaner to simply offer a single slot following the title maybe call it `title-after` and let the consuming app be responsible for rendering the badge/icon if it's needed.